boards/same54-xpro/include/board.h

```diff
@@ -43,6 +44,24 @@ extern "C" {
 #define ATCA_PARAM_I2C           I2C_DEV(1)
 /** @} */
 
+/**
+ * @brief    AT24Mac provides a EUI-48
+ */
+static inline int _at24mac_get_eui48(const void *arg, eui48_t *addr)
+{
+    return at24mac_get_eui48((uintptr_t)arg, addr);
+}
+
+/**
+ * @name    EUI-48 sources on the board
+ *          AT24Mac is present on the board
+ * @{
+ */
+#define EUI48_PROVIDER_FUNC   _at24mac_get_eui48
+#define EUI48_PROVIDER_TYPE   NETDEV_ANY
+#define EUI48_PROVIDER_INDEX  0
```

Comment:
```suggestion

```
I think here it make more sense to leave the overridable default in `eui48_provider_params.h`.